app/scaffold-controllers/cartModalController.js

```diff
@@ -1,45 +1,78 @@
 define([
```

Comment:
open question: how many different cart controllers do we need before we put them in their own folder in `scaffold-controllers`? like `scaffold-controllers/cart`. It's probably okay for now, but I can just see this folder growing pretty large if all of our other components have 2-3 controllers associated with them. thoughts?